README.md

```diff
@@ -15,83 +15,198 @@ npm run test
 
 This project works a bit differently than a regular test suite for now. We have Jest installed more as a test runner, but we don't really use assertions for now. We use it more to gather and report numbers.

```

Comment:
Could you plase add on the above line 9?

```
npm run test-all
```

Reply:
I wanted it to be written in each separate block.
Because "test-all" and "test" are separate methods .